src/main/java/frc/robot/RobotContainer.java

```diff
@@ -548,6 +561,37 @@ private void burnFlashAllSparks() {
     Timer.delay(0.25);
   }
 
+  /**
+   * @return true when the currently prepped state is ready to score
+   */
+  public boolean readyToScoreCheck() {
+    switch (prepState) {
+      case OFF:
+        lights.setLEDColor(LightCode.OFF);
```

Comment:
This will constantly turn off the lights anytime we're in prep state OFF, is that intended?

Reply:
I think so?

Reply:
What about going from home to intaking then back to home?

Reply:
oh

Reply:
So maybe it doesn't get set to anything here, but it gets set to OFF at the end of the selectcommand for scoring?

Reply:
^^ i did this